lib/tpm2_util.h

```diff
@@ -11,7 +11,8 @@
 #define BUFFER_SIZE(type, field) (sizeof((((type *)NULL)->t.field)))
 
 #define TPM2B_TYPE_INIT(type, field) { .t = { .size = BUFFER_SIZE(type, field), }, }
-#define TPM2B_EMPTY_INIT { .t = { .size = 0, }, }
+#define TPM2B_INIT(xsize) { .t = { .size = xsize, }, }
```

Comment:
TPM2B_INIT(xsize) conflicts with the original definition provided in tpm2-tss branch 1.x:
- https://github.com/01org/tpm2-tss/tree/1.x/include/sapi/tpmb.h

```
#define TPM2B_INIT(TYPE, name)					\
    TPM2B_##TYPE    name = {sizeof(name.t.buffer), {0}}
```

In tpm2-tss master branch, "TPM2B_INIT(TYPE, name)" has been removed since 2017-8-15 at https://github.com/01org/tpm2-tss/commit/a3b6e1d54b1005d2e8148609b88955ed3dc4891c
, while 1.x branch still defines "TPM2B_INIT(TYPE, name)"

Reply:
tools/tpm2_pcrevent.c#L94
```
TPM2B_EVENT buffer = TPM2B_INIT(file_size);
```

Reply:
@liuqun What's the effect of this, failing to compile? Maybe we can undef the version of it if it exists.